src/include/rbd/librbd.h

```diff
@@ -1528,6 +1600,37 @@ CEPH_RBD_API int rbd_group_snap_rollback_with_progress(rados_ioctx_t group_p,
                                                        librbd_progress_fn_t cb,
                                                        void *cbdata);
 
+// RBD group mirroring support functions
+CEPH_RBD_API int rbd_mirror_group_list(rados_ioctx_t p, char *names,
+                                       size_t *size);
+CEPH_RBD_API int rbd_mirror_group_enable(rados_ioctx_t p, const char *name,
+                                         rbd_mirror_image_mode_t mirror_image_mode,
+                                         uint32_t flags);
+CEPH_RBD_API int rbd_mirror_group_disable(rados_ioctx_t p, const char *name,
+                                          bool force);
+CEPH_RBD_API int rbd_mirror_group_promote(rados_ioctx_t p,
+                                          const char *name,
+                                          uint32_t flags,
+                                          bool force);
+CEPH_RBD_API int rbd_mirror_group_demote(rados_ioctx_t p,
+                                         const char *name,
+                                         uint32_t flags);
+CEPH_RBD_API int rbd_mirror_group_resync(rados_ioctx_t p, const char *name);
+CEPH_RBD_API int rbd_mirror_group_create_snapshot(rados_ioctx_t p,
```

Comment:
@pkalever I've changed this API as discussed in https://docs.google.com/document/d/1LhCf89z0tIqCJI68pL9UKD9FBSZ80rhOHAZQ120xgCQ/edit?disco=AAABPDYqdL4
Please include this commit https://github.com/ceph/ceph/pull/59773/commits/82f7ecb36a7e1e75fca3601cc886da922f307903 that has implemented these changes.

Reply:
Sure. Noted Ramana. Thanks!